core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,38 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $scope.userIsLoggedIn = GLOBALS.userIsLoggedIn;
+        $scope.activitiesOwned = {explorations: {}, collections: {}};
+        if ($scope.userIsLoggedIn) {
+          $http.get('/creatordashboardhandler/data')
+            .then(function(response) {
+              $scope.libraryGroups.forEach(function(activityGroups) {
+                var activitySummaryDicts =
+                (activityGroups.activity_summary_dicts);
+
+                activitySummaryDicts.forEach(function(activity) {
+                  if (activity.type === 'exploration') {
```

Comment:
'exploration' should be a string constant, ACTIVITY_TYPE_EXPLORATION.

Also, let's do:

if activity.type === ACTIVITY_TYPE_EXPLORATION do ...
else if activity.type === ACTIVITY_TYPE_COLLECTION do ...
else throw Error(...)

Reply:
Umm, why do we need to throw an error? activity type will always be an exploration or collection.